### ISSUE TITLE
elements with no accessible names are responsible for accessibility ranking

### ISSUE THREAD

**Issue by Lalisfeed** [thenewboston-developers/Website#1949]
**Bug Description**
The website has medium accessibility ranking according to the light house report. One of the problem is with no accessible names for the buttons , link and the menu item. 

If these are properly practiced then there might be a chances for improvements of rankings

**Steps to Reproduce**
1. Check lighthouse  results of https://thenewboston.com

**Expected behavior**
Good Accessibility ranking (90 - 100)

**Actual behavior**
low Accessibility ranking (85)

![lig_rep](https://user-images.githubusercontent.com/59479762/125825848-cfdadb58-f21a-4068-97b1-310d9035deeb.png)

**Comment by Lalisfeed** [comment]
results can be checked over [Web.Dev](https://web.dev)

**Comment by Lalisfeed** [comment]
- The navigation menu button for mobile doesn't have an accessible name
- Social media links don't have discernible names
- Background and foreground colors for cards do not have a sufficient contrast ratio.

**Comment by Lalisfeed** [comment]
- Button elements do not have accessible names.

```<div class="Icon FaqDropdownCard__toggle-icon Icon--button" data-testid="Icon" role="button" tabindex="0" style="height: 30px; width: 30px;">```

- menuitem does not have an accessible name

```<button class="TopNavMobileMenu__button">```

- Links do not have a discernible name

```<a class="A SocialMediaIcon HomeHero__SocialMediaLink HomeHero__SocialMediaLink--dis…" data-testid="A" href="https://discord.gg/thenewboston" rel="noreferrer" target="_blank">```

```<a class="A SocialMediaIcon Footer__SocialMediaLink" data-testid="A" href="https://github.com/thenewboston-developers" rel="noreferrer" target="_blank">```

In case of contrast
- Background and foreground colors do not have a sufficient contrast ratio.

```<div class="GetStartedCard__description">```
